### ISSUE TITLE
Android build script

### ISSUE THREAD

**Issue by nandanhere** [maltfield/cross-platform-python-gui#8]
im currently working on the android script to build apks directly. However i am not planning to use the python-for-android method and instead use the buildozer one. is there any advantage to using the former?

**Comment by nandanhere** [comment]
I got the first part of the script right (with buildozer). But it takes 14 mins to execute :0 Also buildozer requires a .spec file so that must be taken care of since adding permissions and filetypes is essential in the process

**Comment by maltfield** [comment]
can you please link to the script that builds android apps using a fork of this repo?

**Comment by nandanhere** [comment]
yes sure! [https://github.com/nandanhere/git-workflow-testing/blob/master/build/android/buildApk.sh](https://github.com/nandanhere/git-workflow-testing/blob/master/build/android/buildApk.sh)
